packages/state-transition/src/index.ts

```diff
@@ -53,7 +53,7 @@ export {
 } from "./cache/effectiveBalanceIncrements.js";
 
 // BeaconChain validation
-export {isValidVoluntaryExit} from "./block/processVoluntaryExit.js";
+export {isValidVoluntaryExit, isValidVoluntaryExitElectra} from "./block/processVoluntaryExit.js";
```

Comment:
those should probably just be a single function with the fork check handled inside the function